po/LINGUAS

```diff
@@ -2,3 +2,4 @@ fr
 it
 ru
 tr
+cs
```

Comment:
please keep this file sorted alphabetically.

Reply:
done